ibis/pandas/execution/temporal.py

```diff
@@ -73,31 +74,31 @@ def convert_to_offset(n):
     return data.apply(convert_to_offset)
 
 
-@execute_node.register(ops.TimestampAdd, datetime.datetime, datetime.timedelta)
+@execute_node.register(ops.TimestampAdd, timestamp_types, timedelta_types)
```

Comment:
If the timestamp and timedelta literals are properly converted to `pd.Timestamp` and `pd.Timedelta` beforehand, then defining an `execute_node` for `BinaryOp` (parent of all temporal binary ops) might be enough - not sure though.

Reply:
Yep, this was related to the perf optimization. I can remove it.